invite/src/invite_bot.ts

```diff
@@ -34,6 +34,12 @@ const FULL_MACRO_REGEX = new RegExp(
   `(?<!\\S)${MACRO_PATTERN} @${USERNAME_PATTERN}`,
   'ig'
 );
+// GitHub expires invitations after one week.
+const EXPIRATION_INTERVAL_MS = 7 * 24 * 60 * 60 * 1000;
+
+function expirationDate(): Date {
+  return new Date(Number(new Date()) - EXPIRATION_INTERVAL_MS - 60 * 60 * 1000);
```

Comment:
Not a fan of the back-and-forth type conversions... how about:

```typescript
const date = new Date();
date.setSeconds(date.getSeconds() - EXPIRATION_INTERVAL_SEC - 60 * 60);
```

(you'll have to change the expiration const to seconds, or use date.setMilliseconds but I don't see why that would be beneficial here)

Additional questions:
* why are you doing `- 60 * 60 * 1000` after EXPIRATION_INTERVAL?
* if it isn't _that_ important to be super-accurate, how about just using `date.setDate(date.getDate() - 7)`
* umm... shouldn't this be `+ EXPIRATION_INTERVAL` instead of minus?

Reply:
Made the changes you suggested. It's subtracting because anything older than 7 days before right now (hence the minus) is expired.